storybook/src/elements/nuxeo-date-picker/nuxeo-date-picker.stories.js

```diff
@@ -1,10 +1,12 @@
 import { storiesOf } from '@storybook/polymer';
+import { text } from '@storybook/addon-knobs';
 import { html } from 'lit-html';
 import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-date-picker';
 
-storiesOf('UI/nuxeo-date-picker', module).addElement('nuxeo-date-picker', ({ knobs }) => {
-  const { value } = knobs();
+storiesOf('UI/nuxeo-date-picker', module).add('nuxeo-date-picker', () => {
```

Comment:
I was trying out this story and it is kind of tricky the UX challenge here...
So if we want to input a date, using `value` knob it may be messy since the user uses one format (eg. `YYYY-MM-DD` and the element uses `MM-DD-YYYY`.

Reply:
changed it to a date knob